test/system/src/main/java/io/pravega/test/system/framework/services/kubernetes/AbstractService.java

```diff
@@ -133,23 +142,28 @@ public String getID() {
         // generate Pravega Spec.
         final Map<String, Object> pravegaPersistentVolumeSpec = getPersistentVolumeClaimSpec("20Gi", "standard");
 
+        String pravegaImg = DOCKER_REGISTRY + PREFIX + "/" + PRAVEGA_IMAGE_NAME;
+
         final Map<String, Object> pravegaSpec = ImmutableMap.<String, Object>builder().put("controllerReplicas", controllerCount)
-                                                                                      .put("segmentStoreReplicas", segmentStoreCount)
-                                                                                      .put("debugLogging", true)
-                                                                                      .put("cacheVolumeClaimTemplate", pravegaPersistentVolumeSpec)
-                                                                                      //.put("options", props.getProperties())
-                                                                                      .put("options", props)
-                                                                                      .put("image",
-                                                                                       getImageSpec(DOCKER_REGISTRY + PREFIX + "/" + PRAVEGA_IMAGE_NAME, PRAVEGA_VERSION))
-                                                                                      .put("tier2", tier2Spec())
-                                                                                      .build();
+                                                                                    .put("segmentStoreReplicas", segmentStoreCount)
+                                                                                    .put("debugLogging", true)
+                                                                                    .put("cacheVolumeClaimTemplate", pravegaPersistentVolumeSpec)
+                                                                                    .put("controllerResources", getResources("2000m", "3Gi", "1000m", "1Gi"))
+                                                                                    .put("segmentStoreResources", getResources("2000m", "5Gi", "1000m", "3Gi"))
+                                                                                    //.put("options", props.getProperties())
+                                                                                    .put("options", props)
+                                                                                    .put("image", ImmutableMap.of("repository", pravegaImg))
```

Comment:
Again, this removes the ability to run system tests with a specific version of pravega, which is not acceptable.  I don't see the need for this change?